File: MicroservicesProject/src/UserManagementService/springboot-app/src/main/java/com/UserManagementService/springbootapp/Requests/RegistrationRequest.java
package com.UserManagementService.springbootapp.Requests;

import lombok.Getter;
import lombok.Setter;

public class RegistrationRequest extends BaseRequest{
    @Getter
    @Setter
    public String Login;

    @Getter
    @Setter
    public String Password;

    @Getter
    @Setter
    public String Email;
}

File: MicroservicesProject/src/WPF_Client/ClientBackLogic/Requests/RegistrationRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClientBackLogic.Requests
{
    public class RegistrationRequest: BaseRequest
    {
        public string Login { get; set; }

        public string Password { get; set; }

        public string Email { get; set; }
    }
}
